<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VierGewinnt</class>
 <widget class="QMainWindow" name="VierGewinnt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VierGewinnt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="feldBesetzen">
        <property name="text">
         <string>Feld besetzen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="feldLabel">
        <property name="text">
         <string>Feld (0-3)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="webserviceBox">
        <property name="text">
         <string>Webservice</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QPushButton" name="beenden">
        <property name="text">
         <string>Beenden</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QPushButton" name="neueSpiel">
        <property name="text">
         <string>Neues Spiel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="minimumSize">
         <size>
          <width>79</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
